src/test/java/org/opensearch/knn/quantization/models/quantizationState/QuantizationStateCacheTests.java

```diff
@@ -42,8 +42,10 @@ public void setThreadPool() {
     }
 
     @After
-    public void terminateThreadPool() {
+    public void stopMaintenance() throws IOException {
```

Comment:
nit: you can use cleanup() or tearDown() as a generic method name for test class cleanup.